src/components/Form/Input/InputFile.tsx

```diff
@@ -27,43 +27,46 @@ function InputFile({
   /**
    * works to put a text as a suffix of the Input box
    */
-  hintText,
+  hintText = 'Subir en formato PDF y firmado por todas las partes (si aplica).',
   /**
    * works to put a custom message to the message when doing dragAnDrop
    */
-  dragMessage,
+  dragMessage = 'Suelta aquí',
   /**
    * Indicates the message that goes in the middle of the box where the files are dropped.
    */
-  boxMessage,
+  boxMessage = 'PDF Peso máximo por archivo 20 MB',
   /**
    * is a callback to listen for input changes from outside the component.
    */
   onChange,
   /**
    * the id works so that the label is able to reference the input
    */
-  id,
+  id = 'upload',
   /**
    * It is a text message that goes on top of the box
    */
-  label,
+  label = 'Drag & drop your files or',
   /**
    * is the message that works to call the Input and with this upload files from a text button
    */
-  labelAction,
+  labelAction = 'browse from your device',
```

Comment:
Mmm i don't know why we have some default messages in english and others in spanish

Reply:
I translate the messages into English.